bom-internal/pom.xml

```diff
@@ -290,7 +290,12 @@ POM as their parent.
             <dependency>
                 <groupId>com.amazonaws</groupId>
                 <artifactId>aws-java-sdk-s3</artifactId>
-                <version>1.12.25</version>
+                <version>1.12.108</version>
```

Comment:
property if the versions are intended to always match

Reply:
Actually, these ones disappear altogether in develop